dashboard-for-desktop/articles/dashboard-designer/provide-data.md

```diff
@@ -19,3 +19,4 @@ This section contains the following topics.
 * [Binding to XPO Data Sources](provide-data/binding-to-XPO-data-sources.md)
```

Comment:
I'll leave a comment on this line since I can't comment on the above lines.

1) Since we're trying to avoid the use of "provide", is it possible to update the title of this page to "Supply Data"?

2) Line 7: The topics in this section describe how to provide data to be visualized in the dashboard.

Can you change "provide" to "supply"?

3) Line 9: This section contains the following topics.

Please add a colon at the end instead of a period.